app/models/custom_label.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class CustomLabel
+  def initialize(attrs)
+    @attrs = attrs
+  end
+
+  def id
+    @attrs['id']
+  end
+
+  def display_label
+    @attrs['displayLabel']
+  end
+
+  def display_on_full_text_finder
+    @attrs['displayOnFullTextFinder']
+  end
+
+  def display_on_publication_finder
+    @attrs['displayOnPublicationFinder']
+  end
+
+  def self.all
+    rmapi_customer_root.all.labels.map { |attrs| new attrs }
+  end
+
+  def self.update(id, attrs)
```

Comment:
~~this model represents a single instance of a custom label, right?  in that case i don't think you need to pass the id here, since the model should already have an id (via the method on line 8).~~